MIGRATION.md

```diff
@@ -330,9 +358,9 @@ npm install webpack@5 --save-dev
 Storybook 6.3 supports Angular 12 out of the box when you install it fresh. However, if you're upgrading your project from a previous version, you'll need to do the following steps to force Storybook to use webpack 5 for building your project:
```

Comment:
As long as we're touching this file...

Ee point people to the "Webpack 5 manager build" heading often, and the directions there are incomplete. To work, you need to _also_ follow the directions under the "Angular 12 upgrade" heading, regardless of which framework they're using.

Can we please re-arrange these sections to avoid that confusion? Can attempt myself, if you'd prefer.

Reply:
would love a hand on that, thanks so much! 🙏 

we probably also need to document webpack5/builders in the main documentation site, and start pointing people there, FWIW

Reply:
we can also do that ☝️ as part of this little fire drill